src/components/Layout.js

```diff
@@ -65,56 +61,66 @@ const Layout = ({ children }) => {
+  const tableOfContent = [
+    { label: t('Home page'), path: '/' },
+    { label: t('Quiz'), path: '/quizz' },
+    { label: t('Observations'), path: '/map' },
+    { label: t('Adaptations'), path: '/adaptations' },
+  ];
   return (
-    <div style={{ minHeight: '100vh', backgroundColor: '#FFF' }} className={classes.root}>
-      <CssBaseline />
-
-      <Drawer
-        className={classes.drawer}
-        variant="permanent"
-        classes={{ paper: classes.drawerPaper }}
-        anchor="left"
+    <ThemeProvider theme={theme}>
+      <Grid
+        container
+        className={classes.root}
+        justify="center"
+        alignItems="center"
+        alignContent="center"
       >
-        {['/', '/quizz', '/map', '/adaptations'].map(path => (
-          <Fab
-            color="primary"
-            aria-label="add"
-            className={classes.fab}
-            key={path}
-            to={`/${i18n.language}${path}`}
-            component={Link}
-          >
-            <EmojiNatureIcon />
-          </Fab>
-        ))}
-      </Drawer>
-
-      <div className={classes.content}>
-        <Header siteTitle={siteMetadata.title} />
-        <main>{children}</main>
-
-        <footer style={{ paddingTop: 10 }}>
-          {t('Last build')} {buildTime}
-          <br />
-          <Location>
-            {({ location: { pathname } }) => (
-              languageIds.map(language => (
-                <span key={language}>
-                  <Link to={adaptPathname(pathname, language)} lang={language}>{language}</Link>
-                  {' '}
-                </span>
-              ))
-            )}
-          </Location>
-        </footer>
-      </div>
-    </div>
+        <CssBaseline />
+
+        <Drawer
+          className={classes.drawer}
+          variant="permanent"
+          classes={{ paper: classes.drawerPaper }}
+          anchor="left"
+        >
+          {tableOfContent.map(({ label, path }) => (
+            <Tooltip title={label} placement="right">
+              <Fab
+                color="primary"
+                aria-label="add"
+                className={classes.fab}
+                key={path}
+                to={`/${i18n.language}${path}`}
+                component={Link}
+                size="small"
+              >
+                <EmojiNatureIcon />
+              </Fab>
+            </Tooltip>
+          ))}
+        </Drawer>
+        <Grid item className={classes.content} xs={12} md={9} xl={8}>
+          <Header
+            siteTitle={siteMetadata.title}
+            parentSite="https://agriadapt.eu/"
+            logo="../images/agriAdapt-logo.svg"
+          />
+          <main>{children}</main>
+
+          <footer style={{ paddingTop: 10 }}>
+            {t('Last build')} {buildTime}
+            <br />
+          </footer>
+        </Grid>
+      </Grid>
+    </ThemeProvider>
   );
 };
 
 Layout.propTypes = {
   children: PropTypes.node.isRequired,
+  theme: PropTypes.node.isRequired,
```

Comment:
Theme should not be provided through props.

If needed, it can be accessed through `const theme = useTheme();`

Reply:
done